README.md

```diff
@@ -1,3 +1,158 @@
+
+## Table of Contents
+
+- [Install](#install)
+  - [npm](#npm)
+  - [Use in Node.js](#use-in-nodejs)
+  - [Use in a browser with browserify, webpack or any other bundler](#use-in-a-browser-with-browserify-webpack-or-any-other-bundler)
+  - [Use in a browser Using a script tag](#use-in-a-browser-using-a-script-tag)
+- [Usage](#usage)
+- [API](#api)
+- [Contribute](#contribute)
+  - [Setup](#setup)
+  - [Test](#test)
+- [License](#license)
+
+## Install
+
+### npm
+
+```sh
+> npm install @fairdatasociety/fair-data-protocol --save
+```
+
+### yarn
+
+```sh
+> yarn add @fairdatasociety/fair-data-protocol
+```
+
+### Use in Node.js
+
+**We require Node.js's version of at least 16.x**
+
+```js
+const FDP = require('@fairdatasociety/fair-data-protocol');
+```
+
+### Use in a browser with browserify, webpack or any other bundler
+
+```js
+const FDP = require('@fairdatasociety/fair-data-protocol');
+```
+
+### Use in a browser Using a script tag
+
+Loading this module through a script tag will make the `fdp` object available in the global namespace.
+
+```html
+<script src="https://unpkg.com/@fairdatasociety/fair-data-protocol/dist/index.browser.min.js"></script>
+```
+
+## Usage
+
+Creating FDP account
+
+```js
+import { FairDataProtocol } from '@fairdatasociety/fair-data-protocol'
+
+const fdp = new FairDataProtocol('http://localhost:1633', 'http://localhost:1635')
+
+const account = await fdp.account.register('myusername', 'mypassword')
+console.log(account) // prints account information
+```
+
+Login with FDP account
+
+```js
+// before login, we need to enter the username of the account and the Ethereum address associated with it
+await fdp.account.setUserAddress('otherusername', '0x.....')
+const wallet = await fdp.account.login('otherusername', 'mypassword')
+console.log(wallet) // prints downloaded and decrypted wallet
+```
+
+Getting list of pods
+
+```js
+const pods = await fdp.personalStorage.list()
+console.log(pods) // prints list of user's pods
+```
+
+## Documentation
+
+You can generate API docs locally with:
+
+```sh
+npm run docs
+```
+
+The generated docs can be viewed in browser by opening `./docs/index.html`
+
+## Contribute
+
+There are some ways you can make this module better:
+
+- Consult our [open issues](https://github.com/fairDataSociety/fair-data-protocol/issues) and take on one of them
+- Help our tests reach 100% coverage!
+
+### Setup
+
+Install project dependencies with
+
+```sh
+npm i
+```
+
+### Test
+
+The tests run in both context: node and dom with Jest.
+
+To run the integration tests, you need to use our [`bee-factory`](https://github.com/fairDataSociety/bee-factory/) project. Clone the repo, you can use our prebuilt Docker images with setting env. variables like:
+
+```bash
+export FAIROS_VERSION="0.7.3"
+export BLOCKCHAIN_VERSION="1.2.0"
+export BEE_ENV_PREFIX="swarm-test"
+export BEE_IMAGE_PREFIX="docker.pkg.github.com/ethersphere/bee-factory"
+```
```

Comment:
These are default env variables that the project contains. I wouldn't be so verbose about it, for more detail they can check out FDP's bee-factory repo